x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/template/IndexTemplateRegistry.java

```diff
@@ -154,7 +154,7 @@ private void addTemplatesIfMissing(ClusterState state) {
             if (creationCheck.compareAndSet(false, true)) {
                 IndexTemplateMetadata currentTemplate = state.metadata().getTemplates().get(templateName);
                 if (Objects.isNull(currentTemplate)) {
-                    logger.debug("adding index template [{}] for [{}], because it doesn't exist", templateName, getOrigin());
+                    logger.info("adding index template [{}] for [{}], because it doesn't exist", templateName, getOrigin());
```

Comment:
I bumped this up to info to better match the `upgrading index template` message. The adding of the template gets logged on the master node (at info level) but having this at `info` level helps to make the upgrade/add consistent.